backend/chromeApiServer/src/main/java/com/canvas/controllers/chromeApiServer/ChromeApiController.java

```diff
@@ -40,25 +40,28 @@ public ChromeApiController(EvaluationService studentEval, CanvasClientService ca
      */
     @GetMapping(
             value = "/execute/courses/{courseId}/assignments/{assignmentId}/submissions/{studentId}",
-            produces = { "application/json" }
+            produces = {"application/json"}
     )
     public ResponseEntity<CommandOutput> initiateInstructorCodeEvaluation(
             @RequestHeader("Authorization") String bearerToken,
             @PathVariable("assignmentId") String assignmentId,
             @PathVariable("courseId") String courseId,
             @PathVariable("studentId") String studentId,
-            @RequestParam("userType") UserType type
+            @RequestParam("userType") String type
```

Comment:
why change this to string? I thought the UserType enum was clean

Reply:
Super valid point. I updated back to UserType since Spring will return a bad request if an ENUM is not passed.

Reply:
i see, i think as long as the param is passed as all caps to the request, it should be interpret it as an enum